### PR TITLE
[PBF-546] Create build scripts for Akamai HD flash plugin.

### PR DESCRIPTION
**Pull request by sarita-ooyala**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -54,15 +59,25 @@ gulp.task('init_module', function(callback) {
   });
 });
 
-gulp.task('build_flash', function(callback) {
-  exec("ant -file build_flash.xml", function(err) {
+gulp.task('build_flash_osmf', function(callback) {
```
Comment (angieisrani) [inline gulpfile.js]: small note that the indentation is a tiny bit confusing here

File: gulpfile.js [hunk 2]
```diff
@@ -54,15 +59,25 @@ gulp.task('init_module', function(callback) {
...
+    exec("ant -buildfile build_flash.xml -Dbuild=build/classes build-CCswc", function(err) {
+    exec("ant -buildfile build_flash.xml -Dbuild=build/classes build-osmf", function(err) {
```
Comment (angieisrani) [inline gulpfile.js]: I don't see a condition for error in the outer function

**Comment by sarita-ooyala** [comment]
All changes are included in PBF-540. Closing the pull request .

**Comment by sarita-ooyala** [comment]
All changes are included in PBF-540. Closing the pull request .

**Pull request closed without merging**